guest-app/gulpfile.js

```diff
@@ -6,7 +6,6 @@ gulp.task('guest-app-home', function () {
         .pipe(webserver({
             fallback:'src/index.html',
             open:true,
-            port: 3000,
```

Comment:
I didn't understand your question properly. Yes, you can use default port number, but add it here, so we can easily change it if we want.